src/OVAL/probes/CMakeLists.txt

```diff
@@ -0,0 +1,241 @@
+add_subdirectory("crapi")
+
+set(PROBE_INSTALL_DIR ${OVAL_PROBE_DIR})
+
+add_library(probe
+    "probe/fini.c"
+    "probe/offline_mode.c"
+    "probe/preload.c"
+    "probe/init.c"
+    "probe/main.c"
+    "probe/input_handler.c"
+    "probe/input_handler.h"
+    "probe/worker.c"
+    "probe/worker.h"
+    "probe/signal_handler.c"
+    "probe/signal_handler.h"
+    "probe/probe.h"
+    "probe/probe.c"
+    "probe/entcmp.c"
+    "probe/entcmp.h"
+    "probe/icache.c"
+    "probe/icache.h"
+    "probe/option.c"
+    "probe/option.h"
```

Comment:
indents :) choose whatever you want but stick to it, the other hunks in the PR are using tabs for indents, this uses 4 spaces